### PR TITLE
Added User-Agent for both requests

### PR DESCRIPTION
**Pull request by croqaz**
Headers look like this:

```
<CIMultiDictProxy('Host': '127.1:8080',
'User-Agent': 'scrapinghub-autoextract/0.1.1 requests/2.22.0',
'Accept-Encoding': 'gzip, deflate',
'Accept': '*/*',
'Connection': 'keep-alive',
'Content-Length': '58',
'Content-Type': 'application/json',
'Authorization': 'Basic ..')>
```

Also added a test for Sync request and a new testing dependency for mocking requests.

### CODE REVIEW

File: autoextract/aio/client.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 from autoextract.utils import chunks
 from .retry import autoextract_retry
 from .errors import ApiError
+from ..utils import user_agent
```
Comment (kmike) [inline autoextract/aio/client.py]: could you please move it to line 14? there is already import from this module

Comment (croqaz) [reply autoextract/aio/client.py]: Sure! I didn't notice. Fixed now

File: autoextract/__version__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__version__ = '0.1.1'
```
Comment (kmike) [inline autoextract/__version__.py]: could you please change it back to 0.1? It is usually better to handle version updates separately from feature pull requests.

Comment (croqaz) [reply autoextract/__version__.py]: Sure! Makes sense. Done

**Comment by kmike** [comment]
Thanks @croqaz for the fix, and for adding tests!

**Pull request merged**